front/app/modules/commercial/native_surveys/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactNode } from 'react';
+import { ModuleConfiguration } from 'utils/moduleUtils';
+import Tab from './admin/components/Tab';
+import { isNilOrError } from 'utils/helperUtils';
+import { IProjectData } from 'services/projects';
+import { IPhaseData } from 'services/phases';
+
+const AdminProjectIdeaEditFormComponent = React.lazy(
+  () => import('./admin/containers/projects/surveys')
+);
+
+type RenderOnHideTabConditionProps = {
+  project: IProjectData;
+  phases: IPhaseData[] | null;
+  children: ReactNode;
+};
+
+const RenderOnHideTabCondition = (props: RenderOnHideTabConditionProps) => {
+  const { project, phases, children } = props;
+  const processType = project.attributes.process_type;
+  const participationMethod = project.attributes.participation_method;
+  const hideTab =
+    (processType === 'continuous' && participationMethod !== 'native_survey') ||
+    (processType === 'timeline' &&
+      !isNilOrError(phases) &&
+      phases.filter((phase) => {
```

Comment:
Isn't it better to use `phases.none` or `phases.every` (and a different condition inside of course) instead of using `filter` and `length === 0`? It expresses intent better. On top of that, I would extract it and give it a good name that expresses what this expression means, like `noNativeSurveysInTimeline`.

Reply:
Nice, fixed. Adding Lucas to review for any other frontend suggestions